### PR TITLE
Correct orientation of masks in R04 and R40 rafts.

### PR DESCRIPTION
**Pull request by jbkalmbach**

### CODE REVIEW

File: doc/versionHistory.rst [hunk 1]
```diff
@@ -6,6 +6,14 @@
 Version History
 ##################
 
+.. _lsst.ts.wep-2.4.5:
```
Comment (suberlak) [inline doc/versionHistory.rst]: Please update that after PR141 gets merged

**Review by suberlak — APPROVED** [review]
Looks good, please wait with merge till PR141 gets merged, then rebase so there's no conflicts, thanks!

**Comment by suberlak** [comment]
Can you post (in the ticket comment) a link to the notebook where you show the improvement or just the images of  how this fix was tested (i.e. before and after ) ?

**Comment by jbkalmbach** [comment]
> Can you post (in the ticket comment) a link to the notebook where you show the improvement or just the images of how this fix was tested (i.e. before and after ) ?

Ticket updated on JIRA

**Pull request merged**